lib/blpapi-wrapper.ts

```diff
@@ -321,23 +348,38 @@ export class Session extends events.EventEmitter {
             if (undefined !== cid) {
                 throw new Error('Subscription already exists for index ' + i);
             }
+
+        }).map((s: Subscription): SubscriptionAndService => {
+            return { subscription: s, serviceUri: getServiceForSecurity(s.security) };
```

Comment:
Style: Please conform to the style guide with respect to [object literals](http://github.com/bloomberg/blpapi-http/blob/6e27373b1b7219fe910ae5ab7879bab235c07aa9/doc/style-guide.md#object-and-array-literals).